docs/src/hohqmesh_tutorial.md

```diff
@@ -0,0 +1,492 @@
+\begin{MODEL}
+
+    \begin{OUTER_BOUNDARY}
+
+       \begin{PARAMETRIC_EQUATION_CURVE}
+          name = OuterCircle
+          xEqn = x(t) = 8.0*sin(2.0*pi*t)
+          yEqn = y(t) = 8.0*cos(2.0*pi*t)
+          zEqn = z(t) = 0.0
+       \end{PARAMETRIC_EQUATION_CURVE}
+
+    \end{OUTER_BOUNDARY}
+
+    \begin{INNER_BOUNDARIES}
+
+       \begin{CHAIN}
+         name = IceCreamCone
+         \begin{END_POINTS_LINE}
+            name = LeftSlant
+            xStart = [-2.0, 1.0, 0.0]
+            xEnd   = [ 0.0, -3.0, 0.0]
+         \end{END_POINTS_LINE}
+
+         \begin{END_POINTS_LINE}
+            name = RightSlant
+            xStart = [ 0.0, -3.0, 0.0]
+            xEnd   = [ 2.0, 1.0, 0.0]
+         \end{END_POINTS_LINE}
+
+         \begin{CIRCULAR_ARC}
+            name        = IceCream
+            units       = degrees
+            center      = [ 0.0, 1.0, 0.0]
+            radius      = 2.0
+            start angle = 0.0
+            end angle   = 180.0
+         \end{CIRCULAR_ARC}
+       \end{CHAIN}
+
+    \end{INNER_BOUNDARIES}
+
+\end{MODEL}
+\end{FILE}
+```
+
+The first alteration is that we have altered the second block of information
+`BACKGOUND_GRID` within the `CONTROL_INPUT` to be
+```
+\begin{BACKGROUND_GRID}
+   background grid size = [1.0, 1.0, 0.0]
+\end{BACKGROUND_GRID}
+```
+Because this mesh control file has an outer boundary that determines the extent of the domain to be meshed.
+Therefore, we only need to supply the `background grid size` to the `BACKGROUND_GRID` control input.
+
+The second alteration is that the `MODEL` now contains information for an `OUTER_BOUNDARY`.
+In this case it is a circle of radius `8` centered at `[0.0, 0.0, 0.0]` written as a set of
+`PARAMETRIC_EQUATION_CURVE`s.
+```
+    \begin{OUTER_BOUNDARY}
+
+       \begin{PARAMETRIC_EQUATION_CURVE}
+          name = OuterCircle
+          xEqn = x(t) = 8.0*sin(2.0*pi*t)
+          yEqn = y(t) = 8.0*cos(2.0*pi*t)
+          zEqn = z(t) = 0.0
+       \end{PARAMETRIC_EQUATION_CURVE}
+
+    \end{OUTER_BOUNDARY}
+```
+Just as with the inner boundary curves, we must assign a name to the `OUTER_BOUNDARY`. It will be included
+in the generated `.mesh` file and is used within the Trixi.jl elixir file to set boundary conditions.
+
+The resulting mesh generated by HOHQMesh.jl is given in the following figure.
+
+![mesh_curved_sides](https://user-images.githubusercontent.com/25242486/129603957-6a92618f-9ed8-4072-b6ab-05533bea746a.png)
+
+Again, the mesh file `ice_cream_curved_sides.mesh` and its associated file for plotting
+`ice_cream_curved_sides.tec` are placed in the `out` folder.
+
+## Running Trixi.jl on `ice_cream_curved_sides.mesh`
+
+We can reuse much of the elixir file to setup the uniform flow over an ice cream cone from the
+previous part of this tutorial. The only components of the elixir file that must be changed are the boundary condition
+dictionary because we now have a boundary named `OuterCirle` instead of four edges of a bounding box.
```

Comment:
```suggestion
previous part of this tutorial. The only component of the elixir file that must be changed is the boundary condition
dictionary because we now have a boundary named `OuterCirle` instead of four edges of a bounding box.
```